### PR TITLE
feat: add Zola

### PR DESCRIPTION
**Pull request by henrifrancois**
Adds support for Zola

### CODE REVIEW

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @henrifrancois!

You should `require` the JSON file in:
https://github.com/netlify/framework-info/blob/4a28a9a9cbf64137a620f8dbe5b560e087e30f0e/src/frameworks/main.js#L29

Also how can we differentiate Zola from Hugo? See:
https://github.com/netlify/framework-info/blob/fb0416a39b7768e1103f3b01be9a272f17c37ce5/src/frameworks/hugo.json#L8

**Comment by henrifrancois** [comment]
After reviewing it seems Zola _only_ uses a `config.toml` for its configuration options. I am unsure if that's a sufficient distinction between them both, however.

File: package-lock.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
         "react-dom": "^17.0.1",
         "test-each": "^2.0.0",
         "tmp-promise": "^3.0.2",
-        "webpack": "^5.41.0",
+        "webpack": "^5.10.1",
```
Comment (erezrokah) [inline package-lock.json:47]: Can we undo this? Seems unrelated to the PR

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks for checking @henrifrancois.

I think this means that sometimes we'll detect both `hugo` and `zola` until we find a better way to do it (I believe we'll need to add support for pattern matching inside configuration files).

**Review by erezrokah — APPROVED** [review]

**Comment by erezrokah** [comment]
Thanks @henrifrancois, I opened https://github.com/netlify/framework-info/issues/329 as a follow up

**Pull request merged**